pkgs/tools/graphics/glee/default.nix

```diff
@@ -0,0 +1,31 @@
+{stdenv, fetchgit, pkgs}:
+
+stdenv.mkDerivation rec {
+  name = "glee";
```

Comment:
The name should contain a version. If you refer to an unreleased commit use the time stamp of that commit.